Content.Server/Explosion/EntitySystems/TriggerSystem.cs

```diff
@@ -189,6 +191,12 @@ private void HandleRattleTrigger(EntityUid uid, RattleComponent component, Trigg
 
         private void OnTriggerCollide(EntityUid uid, TriggerOnCollideComponent component, ref StartCollideEvent args)
         {
+            //Don't trigger if we collide with an entity that's laying down and is not our target.
+            if (TryComp<TargetedProjectileComponent>(uid, out var targetedProjectile) &&
+                args.OtherFixture.CollisionLayer == (int) CollisionGroup.LayingDownMobLayer &&
+                targetedProjectile.Target != args.OtherEntity)
+                return;
+
             if (args.OurFixtureId == component.FixtureID && (!component.IgnoreOtherNonHard || args.OtherFixture.Hard))
                 Trigger(uid);
         }
```

Comment:
Adding specific targeted checks to every collision event is a bad idea.

Reply:
I made an event that decides it for all projectiles, hitscans still have their own logic.